src/hotspot/share/prims/jvmtiRedefineClasses.cpp

```diff
@@ -2188,8 +2192,8 @@ void VM_RedefineClasses::rewrite_cp_refs_in_method(methodHandle method,
     switch (c) {
       case Bytecodes::_ldc:
       {
-        int cp_index = *(bcp + 1);
-        int new_index = find_new_index(cp_index);
+        u1 cp_index = *(bcp + 1);
```

Comment:
This could be a u2 to avoid confusion.  Since it's ldc, the cp_index in the ldc bytecode is only a u1 but this didn't get a Wconversion error so I should probably keep it as int.

Reply:
I agree - using u1 to match the spec is a good thing here.